src/stores/ProjectStore.ts

```diff
@@ -16,7 +16,7 @@ export const useProjectStore = defineStore('project-store', {
     actions: {
         refreshProjectList() {
             const projectService = new ProjectService();
-            projectService.getProjects()
+            projectService.getProjects(100)
```

Comment:
Setting the limit to 100 is not optimal. A better option could be considered here. Lazy loading, infinite scrolling or dynamic loading are concepts that can be investigated here.